### ISSUE TITLE
failureTopic isn't actually optional

### ISSUE THREAD

**Issue by schmod** [awslabs/aws-lambda-redshift-loader#115]
If a locking error is encountered, `index.js` will throw an uncaught `TypeError` if an SNS topic has not been configured for failures. 

The error-logging code near [line 433 of index.js](https://github.com/awslabs/aws-lambda-redshift-loader/blob/d572e4aa16ef68093aaa13dbe468745073038d48/index.js#L433) clearly illustrates the problem:

```js
var e = "Unable to write "
  + itemEntry
  + " in "
  + addFileRetryLimit
  + " attempts. Failing further processing to Batch "
  + thisBatchId
  + " which may be stuck in '"
  + locked
  + "' state. If so, unlock the back using `node unlockBatch.js <batch ID>`, delete the processed file marker with `node processedFiles.js -d <filename>`, and then re-store the file in S3";
console.log(e);
exports.sendSNS(config.failureTopicARN.S, "Lambda Redshift Loader unable to write to Open Pending Batch", e, function() {
    context.done(error, e);
}, function(err) {
    console.log(err);
    context.done(error, "Unable to Send SNS Notification");
});
```

Because `failureTopicARN` isn't always defined, we'll throw an exception while attempting to get a value for `failureTopicARN.S`.

**Comment by IanMeyers** [comment]
Right you are. This is fixed in 2.4.6 https://github.com/awslabs/aws-lambda-redshift-loader/commit/fb3ec3e90f1a755cbd745b5a586eb7605093c562

**Comment by schmod** [comment]
Thanks for the quick fix!
